README.rst

```diff
@@ -28,133 +29,53 @@ Nengo: Large-scale brain modelling in Python
 Installation
 ============
 
-To install Nengo, use::
-
-  pip install nengo
-
-Nengo depends on `NumPy <http://www.numpy.org/>`_.
-If you have difficulty installing,
-try installing NumPy first (see below).
-
-Nengo supports Python 2.7 and 3.3+.
-
-Developer install
------------------
-
-.. code-block:: bash
-
-   git clone https://github.com/nengo/nengo.git
-   cd nengo
-   python setup.py develop --user
-
-If you're using a ``virtualenv``
-(recommended!) then you can omit the ``--user`` flag.
-
-Installing requirements
------------------------
-
-Nengo's main requirement is Numpy
-(and optionally Scipy for better performance).
-Currently, installing Numpy from ``pip`` will install
-an unoptimized (slow) version,
-so it is preferable to install Numpy
-using one of the following methods.
-It is important to install Numpy
-before installing the other requirements.
+If you're on Windows, install NumPy first.
```

Comment:
You may have to do this on Linux (and Mac?) too. I'm not sure how well the pip install version of Numpy works on all setups. Might be good to have some links to resources for all platforms, possibly in a subsection below that we can reference here.